src/app/shared/widgets/prescriptions-list/prescriptions-list.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { SentResourceTypes } from '../../../_constants/common';
+import { SendResourceService } from '../../../_services/send-resource.service';
+
+@Component({
+  selector: 'app-prescriptions-list',
+  templateUrl: './prescriptions-list.component.html',
+  styleUrls: ['./prescriptions-list.component.scss'],
+})
+export class PrescriptionsListComponent implements OnInit, OnDestroy {
+  prescriptions: any[] = [];
+  subscription: Subscription = new Subscription();
+
```

Comment:
Please use the array subscription method that we are using throught the project to unsubscribe : example, like it is done in the case-filters.component.ts

Reply:
Done.